src/Datadog.Trace.AspNet/TracingHttpModule.cs

```diff
@@ -187,5 +188,31 @@ private void OnError(object sender, EventArgs eventArgs)
                 Log.Error(ex, "Datadog ASP.NET HttpModule instrumentation error");
             }
         }
+
+        private void SetStatusCode(Scope scope, int statusCode)
+        {
+            try
+            {
+                if (scope?.Span != null)
+                {
+                    scope.Span.SetTag(Tags.HttpStatusCode, statusCode.ToString());
+
+                    if (statusCode / 100 == 5)
+                    {
+                        // 5xx codes are server-side errors
+                        scope.Span.Error = true;
+
+                        if (scope.Span.GetTag(Tags.ErrorMsg) == null)
+                        {
+                            scope.Span.SetTag(Tags.ErrorMsg, $"Datadog detected StatusCode={statusCode} that is within the range of server errors: 500-599");
```

Comment:
Consider simplifying the message to `HTTP response has status code {statusCode}`.

Reply:
Resolving since we're not going to add to an error message